doc/Language/exceptions.rakudoc

```diff
@@ -255,7 +255,7 @@ Please consult the section on
 L<resuming of exceptions|/language/exceptions#Resuming_of_exceptions>
 for more information on this.
 
-A C<try>-block is a normal block and as such treats its last statement
+A C<try> block is a normal block and as such treats its last statement
```

Comment:
As long as we’re in here: I think “and as such, uses its last statement as its return value.” reads better - if you agree, please add that update.